src/packages/sdk/src/__tests__/getGenerators/getGenerators.test.ts

```diff
@@ -101,25 +101,28 @@ describe('getGenerators', () => {
     generators.forEach((g) => g.stop())
   })
 
-  test.skip('basic - binaryTargets as env var - native', async () => {
-    process.env.BINARY_TARGETS_ENV_VAR_TEST = '["native"]'
-
-    const aliases = {
-      'predefined-generator': {
-        generatorPath: path.join(__dirname, 'generator'),
-        outputPath: __dirname,
-      },
-    }
+  /* eslint-disable jest/no-standalone-expect */
+  const it = (condition: boolean) => (condition ? test : test.skip)
```

Comment:
Hack alarm :p `conditionalTest` maybe instead? And put in some helper file we hoepfully have?

Reply:
Removed in https://github.com/prisma/prisma/pull/7792